src/options.cpp

```diff
@@ -2057,8 +2057,8 @@ void options_manager::add_options_world_default()
        );
 
     add( "INITIAL_DAY", "world_default", translate_marker( "Initial day" ),
-         translate_marker( "How many days into the year the cataclysm occurred.  Day 0 is Spring 1.  Can be overridden by scenarios.  This does not advance food rot or monster evolution." ),
-         0, 999, 60
+         translate_marker( "How many days into the year the cataclysm occurred.  Day 0 is Spring 1. Day -1 randomizes the start date. Can be overridden by scenarios.  This does not advance food rot or monster evolution." ),
```

Comment:
```suggestion
         translate_marker( "How many days into the year the cataclysm occurred.  Day 0 is Spring 1.  Day -1 randomizes the start date. Can be overridden by scenarios.  This does not advance food rot or monster evolution." ),
```
We require two spaces between sentences.